File: solutions/c/pangram/1/pangram.c
#include "pangram.h"

bool is_pangram(const char *sentence)
{
    int counter = 0;
    if (sentence == NULL)
        return (false);
    for (char c = 65; c <= 90; c++)
    {
        for (size_t i = 0; i < strlen(sentence); i++)
        {
            if (sentence[i] == c || sentence[i] == c + 32)
            {
                counter++;
                break ;
            }
        }
    }
    if (counter == 26)
        return (true);
    return (false);
}

File: solutions/csharp/pangram/1/Pangram.cs
using System;

public static class Pangram
{
    public static bool IsPangram(string input)
    {
        int counter = 0;
        for (int c = 65; c <= 90; c++)
        {
            for (int i = 0; i < input.Length; i++)
            {
                if (input[i] == (char)c || input[i] == (char)(c + 32))
                {
                    counter++;
                    break ;
                }
            }
        }
        if (counter == 26)
            return (true);
        return (false);
    }
}
